packages/core/addon/components/navi-visualization-config/apex-pie.js

```diff
@@ -10,5 +10,22 @@
  * />
  */
 import Component from '@glimmer/component';
+import { set, action } from '@ember/object';
+import { copy } from 'ember-copy';
+import { tagName } from '@ember-decorators/component';
 
-export default class NaviVisualizationConfigApexPieComponent extends Component {}
+@tagName('')
```

Comment:
```suggestion
```

This does not apply for glimmer components so this and the import can be removed